custom/care_pathways/templates/care_pathways/partials/report_table.html

```diff
@@ -21,10 +21,10 @@
             <tr>
                 {% for header in report_table.headers %}
                     <th {% if not report_table.pagination.is_on %}data-sort="{{ header.sort_type }}" data-sortdir="{{ header.sort_direction }}"{% endif %} {% if header.css_class %}class="{{ header.css_class }}"{% endif %}>
-                        <i class="icon-white"></i>
+                        <i class="icon-white fa dt-sort-icon"></i>
```

Comment:
this will be a separate cleanup to make sure the sorting works without icon-white in there. The dt-sort-icon is a placeholder but I wanted to do it in a separate PR as it spans across more than just reports.